docs/api.rst

```diff
@@ -19,7 +19,7 @@ Reading
     :toctree: generated/
 
     open_virtual_dataset
-
+    open_virtual_mfdataset
```

Comment:
Note to self: the docs and especially the readme should be rewritten to put this function front and center.